Exec/SCIENCE/rotating_star/initdata.F90

```diff
@@ -0,0 +1,233 @@
+
+module initdata_module
```

Comment:
why is initdata changed in this PR?

Reply:
This is partially because I was switching branches and didn't notice, but also the previous initdata was a .f90 file rather than a .F90 file so it needs updating anyway.